cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -235,6 +235,40 @@ def _add_storage_widgets(self, metrics, storages_list, namespace, dimension_name
             widgets_list.append(graph_widget)
         return widgets_list
 
+    def _add_fsx_widgets(self, storages_list):
+        common_metrics = [
+            new_pcluster_metric(title="Data Read/Write Ops", metrics=["DataReadOperations", "DataWriteOperations"]),
+            new_pcluster_metric(title="Data Read/Write Bytes", metrics=["DataReadBytes", "DataWriteBytes"]),
+        ]
+        lustre_metrics = common_metrics + [
+            new_pcluster_metric(title="Free Data Storage Capacity", metrics=["FreeDataStorageCapacity"])
+        ]
+        ontap_open_zfs_metrics = common_metrics + [
+            new_pcluster_metric(title="Free Data Storage Capacity", metrics=["StorageCapacity", "UsedStorageCapacity"])
```

Comment:
The graph title is does not strictly correspond to the displayed metrics.
According to the title I expect to see a single metric for the free space.
Furthermore, it would be good to show the same information both for FSxLustre and FSx Ontap/OpenZFS.
What about displaying the difference between the two metrics to show the actual free space?